src/main.rs

```diff
@@ -152,14 +152,18 @@ pub fn main() {
         .arg(
             Arg::with_name("optimize")
                 .long("optimize")
-                .help("Enables the Yul optimizer`"),
+                .help("Enables the Yul optimizer`")
+                .possible_values(&["true", "false"])
+                .default_value("true")
+                .use_delimiter(false)
+                .takes_value(true),
         )
         .get_matches();
 
     let input_file = matches.value_of("input").unwrap();
     let output_dir = matches.value_of("output-dir").unwrap();
     let overwrite = matches.is_present("overwrite");
-    let optimize = matches.is_present("optimize");
+    let optimize = matches.value_of("optimize")==Some("true");
```

Comment:
The formatter would like spaces around `==`.

Reply:
Thanks! The formatter is right. It should be with spaces. I corrected it. And updated the docs at https://github.com/mjobuda/fe/blob/master/docs/src/quickstart/installation.md
Should I do another pull request for the updated doc or how is the procedure?